main.scss

```diff
@@ -23,7 +23,10 @@
 @import 'scss/wrapped_general';
 
 @if $o-typography-is-silent == false {
-	@include oFontsIncludeAll();
+
+	@if $o-typography-load-fonts == true {
```

Comment:
Wouldn't it break the silent mode spec? And the same thing is achievable by using o-fonts directly anyway